tools/lkl/lib/virtio_net_linux_fdnet.c

```diff
@@ -47,9 +47,9 @@ static int linux_fdnet_net_tx(struct lkl_netdev *nd,
 
 	do {
 		ret = writev(nd_fdnet->fd, (struct iovec *)iov, cnt);
-	} while (ret == -1 && errno == EINTR);
+	} while (ret == -1 && (errno == EINTR || errno == EAGAIN));
```

Comment:
> I don't think we need this. It makes the net_tx blocking which hangs the whole lkl

hmm, possible.

> The tx polling can handling EAGAIN properly.

I also expected it but `epoll_wait()` blocks after `writev()` notifies EAGAIN, which I'm investigating now how to overcome it.

the EAGAIN was happened from `virtio_write()=>virtio_process_queue()`, ***not*** from  `poll_thread()=>virtio_process_queue()`.  The former path doesn't use epoll_wait() while the latter uses.